Source/RageCommunity.Library/Peds/PedTasks.cs

```diff
@@ -0,0 +1,447 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RageCommunity.Library.Peds
+{
+	/// <summary>
+	/// Courtesy of <a href="https://alloc8or.re/gta5/doc/enums/eTaskTypeIndex.txt">Alloc8or.re</a>
+	/// </summary>
+	public enum PedTask
+	{
+		CTaskHandsUp = 0,
```

Comment:
Why do all values start with a CTask prefix?

Reply:
Because that's how they are named in the list I pulled them from.